source/extensions/filters/common/ext_authz/ext_authz.h

```diff
@@ -27,6 +28,24 @@ enum class CheckStatus {
   Denied
 };
 
+typedef std::vector<std::pair<Http::LowerCaseString, std::string>> KeyValueHeaders;
```

Comment:
This could be refactored with `TestMetadata` to provide a standard type somewhere for header lists.

Reply:
Sorry, I'm not sure what is needed here. Would you mind to give a bit more context?

Reply:
Just introducing this type in a common header to dedupe it with other identical defs.